integrations/overview.mdx

```diff
@@ -0,0 +1,30 @@
+Benji Integrations allow you to connect your payment processor to Benji, enabling seamless loyalty and rewards experiences for your customers.
+
+## Supported Integrations
+| Platform | Status        |
+|----------|--------------|
+| Square   | Available    |
+| Stripe   | Coming soon  |
+| Others   | Planned      |
```

Comment:
Lets already add Shopify, Salesforce, WooCommerce, Adyen.
For Square - lets have 2 rows , one for Square Payment Processing, and one for Square Loyalty Management